groups/chips-uam-infrastructure/templates/chips_uam_user_data.tpl

```diff
@@ -40,4 +40,7 @@ rm -rf /home/ec2-user/uam/master.txt
 #Create a systemd script that will control the startup of webswing including automated restarts for reboots
 aws s3 cp s3://shared-services.eu-west-2.resources.ch.gov.uk/chips/uam/webswing.service /etc/systemd/system/
 systemctl enable webswing.service
-systemctl start webswing.service
\ No newline at end of file
+systemctl start webswing.service
+
+#Cron job to restart webswing service every Monday at 22:00
+00 22 * * 1 systemctl restart webswing > /root/webswing_service_status
```

Comment:
This wont actually set the cron entry, you need to add this command to a file that cron will run.
Either adding it to the root crontab or a file with /etc/cron.weekly.
If you want a specific time then crontab is safest. 
This should work by creating a file then using that file to populate crontab for the root user

```
echo '#### UAM restart job ####' > /root/cronfile
echo '00 22 * * 1 systemctl restart webswing"' >> /root/cronfile
crontab -u root /root/cronfile
```